transmit-lib/src/cmr/transmit/config.clj

```diff
@@ -96,7 +96,7 @@
 (defconfig kms-scheme-override-json
   "Allow for KMS schema urls to be overriden by setting this value to a JSON
    strings such as:
-   {\"platforms\":\"file://frozen/platforms.csv\",
+   {\"platforms\":\"static\",
```

Comment:
Add comment on what `static` means, i.e. this indicates the schema is loaded from CMR source code statically and also where the static files are located.